blockbuster/lanes/base/abci.go

```diff
@@ -0,0 +1,15 @@
+package base
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+func (l *DefaultLane) PrepareLane(sdk.Context, int64, map[string][]byte) ([][]byte, error) {
+	panic("implement me")
+}
+
+func (l *DefaultLane) ProcessLane(sdk.Context, [][]byte) error {
+	panic("implement me")
+}
+
+func (l *DefaultLane) VerifyTx(sdk.Context, sdk.Tx) error {
+	panic("implement me")
+}
```

Comment:
I wonder if these should just be no-ops rather than panic?

Reply:
I implement this in the follow up PR to this one. Idea here is that you can just extend that base implementation and overwrite other things you might have a special use case for (essentially what you described). This should essentially just be analogous to a single, normal app-side mempool.